### PR TITLE
added js

### PR DESCRIPTION
**Pull request by Zweak**
- fixes issue #163

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 - This Github repository explores the website of a Basic Calculator made by Yoshitha Rathnayake.
 - This website shows a preview of my Calculator. 
 - The link for downloading the Calculator app is also available on the website.
-- The programming languages used to build this website are- HTML, CSS and BOOTSTRAP.
+- The programming languages used to build this website are- HTML, CSS and Javascript.
```
Comment (YoshithaRathnayake) [inline README.md:11]: Don't delete the BOOTSTRAP @Zweak 
```suggestion
- The programming languages used to build this website are- HTML, CSS JAVASCRIPT and BOOTSTRAP.
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
Screenshot needed @Zweak

File: README.md (hunk 1 above)
Comment (Zweak) [reply README.md]: > Screenshot needed @Zweak
Capture5

Made another commit, ready for review.

File: README.md [hunk 2]
```diff
@@ -8,14 +8,11 @@
...
 
 <a href="https://www.w3schools.com/html/"><img src="https://img.icons8.com/color/48/000000/html-5--v1.png"/><a> 
-<a href="https://www.w3schools.com/css/"><img src="https://img.icons8.com/color/48/000000/css3.png"/><a>
-<a href="https://www.w3schools.com/bootstrap/"><img src="https://img.icons8.com/color/48/000000/bootstrap.png"/><a>
-<a href="https://www.w3schools.com/python/"><img src= "https://img.icons8.com/color/48/000000/python.png"/><a>
-<a href="https://www.w3schools.com/js/"><img src= "https://img.icons8.com/color/48/javascript.png"/><a>
-<a href="https://www.tutorialspoint.com/yaml/index.htm"><img width="44px" top="0" src="https://upload.wikimedia.org/wikipedia/commons/5/5a/Official_YAML_Logo.svg"/><a>
+<a href="https://www.w3schools.com/css/"><img src="https://img.icons8.com/color/48/000000/css3.png"/><a> <a href="https://www.w3schools.com/js/"><img src= "https://img.icons8.com/color/48/javascript.png"/><a> 
+<a href="https://www.w3schools.com/bootstrap/"><img src="https://img.icons8.com/color/48/000000/bootstrap.png"/><a> <a href="https://www.tutorialspoint.com/yaml/index.htm"> <img width="41px" height="46px" src="./assets/img/yaml.svg"/><a>
```
Comment (YoshithaRathnayake) [inline README.md:15]: Remove these yaml logo commits @Zweak

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -8,14 +8,11 @@
 - This Github repository explores the website of a Basic Calculator made by Yoshitha Rathnayake.
 - This website shows a preview of my Calculator. 
 - The link for downloading the Calculator app is also available on the website.
-- The programming languages used to build this website are- HTML, CSS and BOOTSTRAP.
+- The programming languages used to build this website are- HTML, CSS, JAVASCRIPT and BOOTSTRAP.
 
 <a href="https://www.w3schools.com/html/"><img src="https://img.icons8.com/color/48/000000/html-5--v1.png"/><a> 
-<a href="https://www.w3schools.com/css/"><img src="https://img.icons8.com/color/48/000000/css3.png"/><a>
+<a href="https://www.w3schools.com/css/"><img src="https://img.icons8.com/color/48/000000/css3.png"/><a> <a href="https://www.w3schools.com/js/"><img src= "https://img.icons8.com/color/48/javascript.png"/><a>
```
Comment (YoshithaRathnayake) [inline README.md:14]: Remove this JS logo also @Zweak only name requested

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (Zweak) [reply README.md]: @YoshithaRathnayake done

File: README.md [hunk 4]
```diff
@@ -8,14 +8,11 @@
 - This Github repository explores the website of a Basic Calculator made by Yoshitha Rathnayake.
 - This website shows a preview of my Calculator. 
 - The link for downloading the Calculator app is also available on the website.
-- The programming languages used to build this website are- HTML, CSS and BOOTSTRAP.
+- The programming languages used to build this website are- HTML, CSS, JAVASCRIPT and BOOTSTRAP.
 
 <a href="https://www.w3schools.com/html/"><img src="https://img.icons8.com/color/48/000000/html-5--v1.png"/><a> 
 <a href="https://www.w3schools.com/css/"><img src="https://img.icons8.com/color/48/000000/css3.png"/><a>
 <a href="https://www.w3schools.com/bootstrap/"><img src="https://img.icons8.com/color/48/000000/bootstrap.png"/><a>
-<a href="https://www.w3schools.com/python/"><img src= "https://img.icons8.com/color/48/000000/python.png"/><a>
```
Comment (YoshithaRathnayake) [inline README.md:16]: Don't remove the python logo @Zweak

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by Zweak** [comment]
@YoshithaRathnayake Can you merge? It's done.

**Pull request closed without merging**